_apidocs/searchgov-results.md

```diff
@@ -64,6 +64,7 @@ Sites indexed via sitemaps or crawling will use the `/i14y` endpoint. Because mo
   | limit (optional)                | Defines the number of results to return. The default is 20, but you can specify between 1 and 50 results. <br> Example: `limit=5`
   | offset (optional)               | Defines the number of results you want to skip from the first result. The offset is used for implementing pagination. The default is 0 and the maximum is 999. <br> Example: `offset=20`
   | sort\_by (optional)             | Allowed variables are date and relevance. The default sort is relevance. Add `sort_by=date` to sort by date.
+  | sitelimit (optional)             | Limits the results to the subdomains and/or subfolders provided. Multiple values can be passed using a space-separated list. Example: `sitelimit=search.gov/about search.gov/get-started`
```

Comment:
@dawnpm That is the two value example - search.gov/about and search.gov/get-started. Per Lora, it needs to be space-separated, but + might work too, I can test if that's our preference.

Reply:
Making it `+` separated introduces some encoding issues, I would rather we keep it space-separated as the API expects if that works.

Reply:
By URL request string, do you mean the hosted SERP url? It accepts a space (https://find.search.gov/search?utf8=%E2%9C%93&affiliate=usasearch&query=rss&sitelimit=search.gov/admin-center%20search.gov/blog).

The code is looking for a space-separated list. Input forms will typically encode the space into a `+` (ex. when you enter a multi-word search term), but since this is making a direct request to an API, I don't think that extra step is necessary. 

As far as clarification goes, what kind of language are you thinking?

Reply:
hm, ok. In this case, let's not add language.